src/modules/hub.bicep

```diff
@@ -31,9 +31,17 @@ var resourceTags = union(tags, {
 
 @description('Optional. Enable telemetry to track anonymous module usage trends, monitor for bugs, and improve future releases.')
 param enableDefaultTelemetry bool = true
+
+@description('Optional. List of scope IDs to create exports for.')
+param exportScopes array
+
 // The last segment of the telemetryId is used to identify this module
 var telemetryId = '00f120b5-2007-6120-0000-40b000000000'
 var finOpsToolkitVersion = '0.0.1'
+var containerNames = ['config', 'ms-cm-exports', 'ingestion']
```

Comment:
nit: Can we move the container vars above the telemetry vars and add a space between?